### PR TITLE
MEB-50: Redirecting about/customers to /supporters page

### PR DESCRIPTION
**Pull request by dpmittal**
/about/customers link is a dead link and it should be redirected to /supporters link 

Issue Link : https://tickets.metabrainz.org/browse/MEB-50

### CODE REVIEW

File: metabrainz/views.py [hunk 1]
```diff
@@ -49,3 +49,7 @@ def bad_customers():
 @index_bp.route('/privacy')
 def privacy_policy():
     return render_template('index/privacy.html')
+
+@index_bp.route('/about/customers')
```
Comment (uklauer) [inline metabrainz/views.py]: The old URL was `/about/customers.html`, so it seems you are missing the `.html` here.

File: metabrainz/views.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from flask import Blueprint, render_template
+from flask import Blueprint, render_template ,redirect , url_for
```
Comment (uklauer) [inline metabrainz/views.py]: The spaces are all over the place here. It should be `foo, bar` – no space before the comma, one space after it, just as in English text.

File: metabrainz/views.py [hunk 3]
```diff
@@ -21,12 +21,12 @@ def about():
...
 def projects():
     return render_template('index/projects.html')
-    
+
 
 @index_bp.route('/team')
 def team():
     return render_template('index/team.html')
-    
+
```
Comment (uklauer) [inline metabrainz/views.py]: These spaces didn’t belong here so it was right to remove them, but that has nothing to do with the redirect issue. Therefore, these indentation fixes should be in a separate commit.

**Comment by dpmittal** [comment]
@uklauer  I have made the changes

**Comment by uklauer** [comment]
I won’t judge whether it is better to remove the indented lines altogether or just the indentation, but in any case, my point about doing it in a separate commit still stands.

**Comment by gentlecat** [comment]
@brainzbot, add to whitelist.

**Comment by dpmittal** [comment]
@uklauer I have fixed the indentation and implemented the redirect feature in two separate commits as you said and @gentlecat I am actually new to flask so I dont know how to write tests at present but I will learn it and will surely send a PR afterwards :)

**Comment by dpmittal** [comment]
@gentlecat I have added the test in a new commit :)
